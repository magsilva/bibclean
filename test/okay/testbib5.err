%% testbib5.bib:2:Unexpected quote usage in German-style braced string.
%% testbib5.bib:14:Unexpected quote usage in German-style braced string.
