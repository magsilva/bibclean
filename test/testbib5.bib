@misc{label,
 abstract = {\TeX{} ben"otigt eine gro"se Anzahl von Hilfskomponenten
                  (Dateien und Programme), deren Bedeutung und
                  gegenseitige \verb="abc"=},
 abstract2 = {\TeX{} ben"otigt eine gro"se Anzahl von Hilfskomponenten
                  (Dateien und Programme), deren Bedeutung und
                  gegenseitige {\verb="abc"=}},
 note = {\TeX{} ben{\"o}tigt eine gro{\"s}e Anzahl von Hilfskomponenten
                  (Dateien und Programme), deren Bedeutung und
                  gegenseitige {\tt"}\verb=abc={\tt"}},
 note2 = {\TeX{} ben{\"o}tigt eine gro{\"s}e Anzahl von Hilfskomponenten
                  (Dateien und Programme), deren Bedeutung und
                  gegenseitige {\verb="abc"=}},
 verbatim1 = {\verb="abc"},
 verbatim2 = {{\verb="abc"}},
 verbatim3 = "{\verb="abc"}",
 everything1 = {umlaut accents: "a, "e, "i, "o, "u, "A, "E, "I, "O, "U,\\
                sharp s: "s,\\
                German quotes: "` (left), "' (right),\\
		French quotes: "< (left), "> (right),\\
                discretionary hyphens: ba"cken, Bet"tuch, xx"ffxx\\
		ligature separator: Auf"|lage,\\
                super discretionary hyphen: Rechen"-fehler,\\
                invisible hyphen: /usr""/spool""/news},
 everything2 = "{umlaut accents: "a, "e, "i, "o, "u, "A, "E, "I, "O, "U,\\
                sharp s: "s,\\
                German quotes: "` (left), "' (right),\\
		French quotes: "< (left), "> (right),\\
                discretionary hyphens: ba"cken, Bet"tuch, xx"ffxx\\
		ligature separator: Auf"|lage,\\
                super discretionary hyphen: Rechen"-fehler,\\
                invisible hyphen: /usr""/spool""/news}",
}
