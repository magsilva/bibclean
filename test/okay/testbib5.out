@Misc{label,
  abstract =     "\TeX{} ben{"o}tigt eine gro{"s}e Anzahl von
                 Hilfskomponenten (Dateien und Programme), deren
                 Bedeutung und gegenseitige \verb={"a}bc{"}=",
  abstract2 =    "\TeX{} ben{"o}tigt eine gro{"s}e Anzahl von
                 Hilfskomponenten (Dateien und Programme), deren
                 Bedeutung und gegenseitige {\verb="abc"=}",
  note =         "\TeX{} ben{\"o}tigt eine gro{\"s}e Anzahl von
                 Hilfskomponenten (Dateien und Programme), deren
                 Bedeutung und gegenseitige {\tt"}\verb=abc={\tt"}",
  note2 =        "\TeX{} ben{\"o}tigt eine gro{\"s}e Anzahl von
                 Hilfskomponenten (Dateien und Programme), deren
                 Bedeutung und gegenseitige {\verb="abc"=}",
  verbatim1 =    "\verb={"a}bc{"}",
  verbatim2 =    "{\verb="abc"}",
  verbatim3 =    "{\verb="abc"}",
  everything1 =  "umlaut accents: {"a}, {"e}, {"i}, {"o}, {"u}, {"A},
                 {"E}, {"I}, {"O}, {"U},\\
                 sharp s: {"s},\\
                 German quotes: {"`} (left), {"'} (right),\\
                 French quotes: {"<} (left), {">} (right),\\
                 discretionary hyphens: ba{"c}ken, Bet{"t}uch,
                 xx{"f}fxx\\
                 ligature separator: Auf{"|}lage,\\
                 super discretionary hyphen: Rechen{"-}fehler,\\
                 invisible hyphen: /usr{""}/spool{""}/news",
  everything2 =  "{umlaut accents: "a, "e, "i, "o, "u, "A, "E, "I, "O,
                 "U,\\
                 sharp s: "s,\\
                 German quotes: "` (left), "' (right),\\
                 French quotes: "< (left), "> (right),\\
                 discretionary hyphens: ba"cken, Bet"tuch, xx"ffxx\\
                 ligature separator: Auf"|lage,\\
                 super discretionary hyphen: Rechen"-fehler,\\
                 invisible hyphen: /usr""/spool""/news}",
}
